App.js

```diff
@@ -1,10 +1,24 @@
 import React, {Component} from 'react';
-import Login from './src/components/Login';
+import {StatusBar} from 'react-native';
+import Home from './src/components/Home';
+import Routes from './src/Routes';
+import { View, StyleSheet } from 'react-native';
 class App extends Component{
       render(){
         return(
-          <Login />
+          <Home />
```

Comment:
Always go back to master before creating a new branch. Otherwise, your pull request will include unrelated changes